packages/guui/htmlTemplate.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import resetCSS from './lib/reset-css';
+import { hashedPath, staticPath } from './lib/asset-path';
+
+export default ({
+    title = 'The Guardian',
+    vendor = ['javascript', 'vendor.js'],
+    fonts = ['css', 'fonts.css'],
+    bundle,
+    css,
+    html,
+    data,
+    cssIDs,
+    nonBlockingJS = '',
+}: {
+    title?: string,
+    vendor?: Array<string>,
+    fonts?: Array<string>,
+    bundle: Array<string>,
+    css: string,
+    html: string,
+    data: {
+        page: string,
+        site: string,
+    },
+    cssIDs: Array<string>,
+    nonBlockingJS?: string,
+}) => {
+    const vendorPath = hashedPath(...vendor);
+    const fontsPath = staticPath(...fonts);
+    const bundlePath = hashedPath(...bundle);
+    const sanitiseDomRefs = jsString =>
```

Comment:
this feels dotcom-specific?

Reply:
hmm OK, I could move `sanitiseDomRefs` into `dotcom/document.js` and sanitise the html returned by `htmlTemplate` in there?